externs/lcevc.js

```diff
@@ -0,0 +1,93 @@
+/**
+* @fileoverview Externs for LcevcDil
+* compiler.
+*
+* @externs
+*/
+'use strict';
```

Comment:
This doesn't do anything in externs AFAIK.  They are only processed by the compiler for type info, not executed or included in compiled output.

Reply:
This is removed.